### DISCUSSION TITLE
[A-L] (2023/24) Foglio 3 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#186] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-10-22 204516](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/aecb6bfe-4e2a-4eaf-abdc-e3dd43aa64cc)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione confermata in classe
- $[8]$ è invertibile in $\mathbb{Z}_{385}\iff\exists\ x: x\cdot [8]=1(mod\ 385)\iff (8,385)|1$

Allora calcoliamo $(385,8)$ con l'algoritmo di Euclide:
$385=8\cdot 48 + 1$
$8= 1 \cdot 8 + 0$
$\Rightarrow (385,8)=1$, $1|1$ allora $[8]$ è invertibile in $\mathbb{Z}_{385}$

Indichiamo con $x=8^{-1}$ l'inverso di $8$:
Per la definizione di congruenza $8\cdot 8^{-1}\equiv 1(mod\ 385)\iff 8\cdot 8^{-1}-1=385y\rightarrow 8\cdot x-385y=1$
Quindi ci basta calcolare l'identità di Bezout per $8x-385y$: $(x,y)=(-48,-1)$ (ovvia dall'algoritmo).
Quindi $[-48]$ è l'inverso di $[8]$ in $\mathbb{Z}_{385}$.

Adesso dobbiamo risolvere l'eq congruenziale $8X\equiv 3(mod\ 385)$:
Abbiamo già detto che $(8,385)=1$, che divide $3$ quindi abbiamo soluzione unica (coprimi)
Moltiplico entrambi i lati per l'inverso trovato $[-48][8][X]\equiv [3][-48](mod\ 385)$ .
$\rightarrow [X]= [-144]\_{385}=[241]\_{385}$ che è soluzione unica in $\mathbb{Z}\_{385}$
Mentre tutte le soluzioni in $\mathbb{Z}$ sono $\\{241+385k\\}$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/47e44b00-2706-4813-8c99-5badd02a1d87)

**Comment by daromacarlo** [comment]
![Immagine 2023-12-12 120600](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/96d87e12-8c33-4e50-ac13-693d19e77a5e)
